frontend/public/components/import-yaml.tsx

```diff
@@ -12,10 +13,15 @@ export const ImportYamlPage = () => {
       </Helmet>
       <AsyncComponent
         loader={() => import('./droppable-edit-yaml').then((c) => c.DroppableEditYAML)}
+        allowMultiple={true}
```

Comment:
```suggestion
        allowMultiple
```

Reply:
done and removing location, no longer needed here